src/golang/cmd/server/handler/get_artifact_result.go

```diff
@@ -55,9 +55,10 @@ type artifactResultMetadata struct {
 	// `status` in the UI.
 	Status            shared.ExecutionStatus            `json:"status"`
 	ExecState         shared.ExecutionState             `json:"exec_state"`
-	Schema            []map[string]string               `json:"schema"`
-	SerializationType artifact_result.SerializationType `json:"serialization_type"`
-	ArtifactType      artifact.Type                     `json:"artifact_type"`
+	Schema            []map[string]string               `json:"schema,omitempty"`
```

Comment:
Quick question about this omitempty keyword. Does it mean that if schema or any of these keys are null that the value will not come back in the response?

Might be easier for me on the frontend to have the keys always available in the response and just get a null value in the payload.